.github/workflows/tests.yml

```diff
@@ -90,7 +90,6 @@ jobs:
     - name: Test `fail-after` is triggered
       id: test_fail-after
       uses: ./.github/actions/testing
-      continue-on-error: true
```

Comment:
this was wrong, since it applies to all steps in the composite action that it's calling